Cargo.toml

```diff
@@ -55,7 +55,6 @@ optional=true
 
 [dependencies.regex]
 version = "1.3"
```

Comment:
For consistency's sake, it might be best to move this up with all of the other key value pairs under `[dependencies]`.

Reply:
Resolved.